samples/gen-js/test/test.js

```diff
@@ -0,0 +1,35 @@
+const assert = require("assert");
```

Comment:
we're still thinking of writing some tests where we don't mock the server along with these, right?